internal/store/store_test.go

```diff
@@ -1,15 +1,195 @@
 package store
 
 import (
+	"github.com/osbuild/osbuild-composer/internal/blueprint"
```

Comment:
Imports are not grouped properly (no big issue)

Reply:
Oh, do we have some rules for this or are there any common rules for this? I always just rely on `go fmt` to do all this work.

Reply:
We try sorting the imports like this:

1) standard library
2) 3rd party packages
3) our packages

However, we don't enforce it (there's sadly no way to enforce it on the CI).

Reply:
Now that I know this, I can enforce it on me myself :D ... Is it correct now?

Reply:
Can `prepare-source` / `go fmt` take care of this somehow ?

Reply:
> However, we don't enforce it (there's sadly no way to enforce it on the CI).

@ondrejbudai there must be linters for that, right? At least in Python both pylint (and I think) isort follow the same policy as you described above.

Reply:
There's `goimports`. I tried it and it had very strange results in some cases.

Then there's [impi](https://github.com/pavius/impi/), but I wasn't brave enough to use it, it doesn't seem much popular. However, I think it does the grouping right. Feel free to investigate it if you see a value in it.